packages/components/tests/dummy/app/templates/components/power-select.hbs

```diff
@@ -0,0 +1,107 @@
+{{page-title "PowerSelect Component"}}
+
+<h2 class="dummy-h2">PowerSelect</h2>
+
+<section>
+  <h3 class="dummy-h3" id="overview"><a href="#overview" class="dummy-link-section">§</a> Overview</h3>
+
+  <p class="dummy-paragraph">
+    A showcase with custom styles applied as overrides to the
+    <a target="_blank" rel="noopener noreferrer" href="https://ember-power-select.com/">ember-power-select addon</a>.
+  </p>
+
+</section>
+
+<section data-test-percy>
+  <h3 class="dummy-h3" id="showcase"><a href="#showcase" class="dummy-link-section">§</a> Showcase</h3>
```

Comment:
Probably before the "Showcase" I would add a "How to use" section in which:
- explain that these styles assume you use ember-power-select with the default theme (see [comment here](https://github.com/hashicorp/design-system/pull/539#discussion_r951247990))
- explain how to use the specific `.hds-power-select` class (initially I thought it should be applied to the component itself, in reality it needs to be applied to a wrapping DOM node).
- explain how to use the `.hds-power-select_after-options`class (move the explanation from the showcase to the howto)